### PR TITLE
Connectionmanager state

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

**Comment by palango** [comment]
Fixed some typos and rebased.

File: raiden/connection_manager.py [hunk 1]
```diff
@@ -144,9 +167,9 @@ def close_all(self, only_receiving=True):
             channels_to_close = (
                 self.receiving_channels[:] if only_receiving else self.open_channels[:]
             )
-            for channel in channels_to_close:
+            for channel_ in channels_to_close:
```
Comment (LefterisJP) [inline raiden/connection_manager.py]: What's the underscore here for?

File: raiden/connection_manager.py [hunk 2]
```diff
@@ -164,7 +187,7 @@ def wait_for_settle(self, closed_channels):
 
     def join_channel(self, partner_address, partner_deposit):
         """Will be called, when we were selected as channel partner by another
-        node. It will fund the channel with up to the partner's deposit, but
+        node. It will fund the channel with up to the partners deposit, but
```
Comment (LefterisJP) [inline raiden/connection_manager.py]: ehm @palango This was not a typo. It was fine before.

Comment (palango) [reply raiden/connection_manager.py]: Are you sure? Why is this right here?

Comment (LefterisJP) [reply raiden/connection_manager.py]: Possessive case in English. A form of the genitive case used in a genitive-noun relationship.

You put the `'s` at the end to indicate posession. In our case to indicate that the deposit belongs to the partner.
There are some exceptions to this rule but they don't apply here.

You could maintain the same semantics by using genitive with composition and turn it into:

> with up to the deposit **of** the partner

tl;dr: Yes I am sure.

Comment (palango) [reply raiden/connection_manager.py]: Good to know.

File: raiden/connection_manager.py (hunk 1 above)
Comment (hackaugusto) [reply raiden/connection_manager.py]: name collision

Comment (LefterisJP) [reply raiden/connection_manager.py]: Yep but with which channel variable? From an initial look at it I could not figure it out.

Comment (hackaugusto) [reply raiden/connection_manager.py]: in a previous version the channel module was imported
